SuperBuild/External_zlib.cmake

```diff
@@ -27,13 +27,13 @@ if(NOT DEFINED ZLIB_ROOT AND NOT Slicer_USE_SYSTEM_${proj})
 
   ExternalProject_SetIfNotDefined(
     Slicer_${proj}_GIT_REPOSITORY
-    "${EP_GIT_PROTOCOL}://github.com/commontk/zlib.git"
+    "${EP_GIT_PROTOCOL}://github.com/jamesobutler/zlib-ng.git"
```

Comment:
Should actually considering using a fork within the Slicer GitHub organization prior to integrating this work

Reply:
See https://github.com/Slicer/zlib-ng